jdaviz/app.py

```diff
@@ -1661,7 +1661,7 @@ def _on_new_viewer(self, msg, vid=None, name=None):
             viewer.state.x_att = x
 
         # Create the viewer item dictionary
-        if name is None:
+        if name is None and self.config != 'imviz':
             name = viewer.__class__.__name__
         new_viewer_item = self._create_viewer_item(
             viewer=viewer, vid=vid, name=name, reference=name
```

Comment:
was this originally added for cubeviz viewer tabs?

Reply:
Yeah... see https://github.com/spacetelescope/jdaviz/issues/1919#issuecomment-1349906208 and https://github.com/spacetelescope/jdaviz/pull/1741#issuecomment-1349914985

Reply:
Either we can patch up the viewer dropdown to account for this, modify PR 1741 solution to pass in ID as reference, or revert PR 1741 completely. I am unsure which is best. Maybe we should discuss.

Reply:
At first thought, I like the idea of using `vid` for name/reference in Imviz - but let's definitely discuss and test if there are any downsides to that approach!